torchgeo/datasets/patternnet.py

```diff
@@ -86,6 +83,7 @@ def __init__(
         self,
         root: str = "data",
         transforms: Optional[Callable[[Dict[str, Tensor]], Dict[str, Tensor]]] = None,
+        loader: Optional[Callable[[str], Any]] = None,
```

Comment:
This shouldn't be an option, there is only one loader that could work for these images. The `__getitem__` will always return Tensors, so it doesn't matter what loader is used as long as it works.

Reply:
Ahh thanks for catching that. I guess my thoughts were along the lines of: If you have a multispectral dataset like EuroSAT, you would want to set a custom loader which can load the geotiffs like below. But you're right in that we should just hardcode a multispectral loader when calling `super().__init__()` so the user doesn't need to worry about it.

```python
def rasterio_loader(path: str) -> np.ndarray:
    with rasterio.open(path, "r") as f:
        x = f.read()
        # VisionClassificationDataset expects images returned with channels last (HWC)
        x = x.tranpose((1, 2, 0))
        return x
```

Reply:
Removed loader arg from datasets.